scripts/generators/base_generator.py

```diff
@@ -125,8 +125,9 @@ def beginFile(self, genOpts):
         for platform in self.registry.tree.findall('platforms/platform'):
             self.vk.platforms[platform.get('name')] = platform.get('protect')
 
-        for tag in self.registry.tree.findall('tags'):
-            self.vk.vendorTags.append(tag.get('name'))
+        for tags in self.registry.tree.findall('tags'):
+            for tag in tags.findall('tag'):
```

Comment:
clearly this was not being used before... while we have this small time of divergence (while I get this file in the Vulkan-Headers), can you apply this fix in the VVL as well :pray:

Reply:
You could say,
*puts on sunglasses*
I already did.

Reply:
https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/6611